apis/v1alpha1/validation.go

```diff
@@ -31,5 +31,8 @@ func ValidateResourceMapping(m *ResourceMapping) (vErr error) {
 	if m.Resource.Provider == "" {
 		vErr = errors.Join(vErr, fmt.Errorf("empty resource provider"))
 	}
+	if _, ok := m.Annotations.AsMap()["assetInfo"]; ok {
```

Comment:
Probably in this package - Can we define the well-known keys as (exposed) consts?